src/group/libp2p_group_mgr.erl

```diff
@@ -0,0 +1,106 @@
+-module(libp2p_group_mgr).
+
+-behaviour(gen_server).
+
+%% API
+-export([
+         start_link/1,
+         mgr/1,
+         add_group/5,
+         remove_group/3,
+         force_gc/1
+        ]).
+
+%% gen_server callbacks
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2,
+         terminate/2, code_change/3]).
+
+-define(SERVER, ?MODULE).
+
+-record(state, {}).
+
+%%%===================================================================
+%%% API
+%%%===================================================================
+
+start_link(TID) ->
+    gen_server:start_link(?MODULE, [TID], []).
+
+mgr(TID) ->
```

Comment:
Maybe put that and `libp2p_config`. As all the other ets related stuff are in there

Reply:
I'm not sure that that's the right place for this, I'm just doing what the group server does.